TestModels/Errors/src/WrappedSimpleErrorsImpl.dfy

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 include "../Model/SimpleErrorsTypesWrapped.dfy"
 
-module {:extern "simple.errors.internaldafny.wrapped"} WrappedSimpleErrorsService refines WrappedAbstractSimpleErrorsService {
```

Comment:
nit/question: did we decide how we're resolving the lack of extern trait for non-Python (and Go?) runtimes in 4.7?

Reply:
José is actively looking into this. I think the current plan is something like
```
module {:extern } WrappedSimpleErrorsService
```
but we're going to work together to see what makes sense over the next week or so. 

My plan is to have him resolve this on main for Java/NET, then I will pull in whatever the solution is.